Business Finder/Model.h

```diff
@@ -0,0 +1,28 @@
+//
+//  Model.h
+//  Business Finder
+//
+//  Created by Michael Johnson on 8/2/16.
+//  Copyright © 2016 Michael Johnson. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "LocationGateway.h"
+#import "FourSquareGateway.h"
```

Comment:
You shouldn't be importing `.h` files into another `.h` file unless you declare a protocol on that `.h` file that you must use. Instead, `@class FourSquareGateway;`